ipatests/prci_definitions/nightly_latest_pki.yaml

```diff
@@ -75,7 +75,7 @@ jobs:
         update_packages: True
         test_suite: test_integration/test_external_ca.py::TestSelfExternalSelf test_integration/test_external_ca.py::TestExternalCAInstall
         template: *pki-master-latest
-        timeout: 3600
+        timeou/ 3600
```

Comment:
typo in timeout

Reply:
Also could you add a description in the commit as to why this change has been made.

Reply:
@ssidhaye Done, and sorry for typo's.